backend/src/org/akvo/lumen/endpoint/dashboard.clj

```diff
@@ -1,183 +1,33 @@
 (ns org.akvo.lumen.endpoint.dashboard
   (:require [compojure.core :refer :all]
-            [clojure.java.jdbc :as jdbc]
-            [hugsql.core :as hugsql]
             [org.akvo.lumen.component.tenant-manager :refer [connection]]
-            [org.akvo.lumen.util :refer [squuid]]
-            [ring.util.response :as resp]))
+            [org.akvo.lumen.lib.dashboard :as dashboard]
```

Comment:
I'm not sure I have a good answer to this. Domain looked odd to me. Suggesting that files outside of that was not the domain or that the domain dir contained different domains, like endpoints or components. 

I could argue that we have domain logic the files at src/org/akvo/lumen/* & in components to. But maybe auth.clj, import.clj, job-execution.clj & transformation.clj should move into the domain logic dir (currently lib) to. 

So in short I don't have a good answer, and I'm not married to "lib" I was actually considering "lumen" and "api" to. But i would be fine with something else.